src/main/kotlin/rat/poison/scripts/BombTimer.kt

```diff
@@ -76,9 +78,21 @@ fun bombUpdater() = every(25, true) {
             canDefuse = false
             gettingDefused = false
         }
+        var b: Boolean? = false
+        if (curSettings["LS_BOMB"].strToBool() && me.team() == 3.toLong() && !me.hasDefuser() && bombState.timeLeftToExplode < 10.2 && b == false)
+            run {
+                CSGO.clientDLL[dwUse] = 5
```

Comment:
It will do nothing, since pressing E will just be a +use. However, releasing it will be a -use, which will cause it to lose the bomb defuse. So if the user presses it, they will have to hold it until the bomb defuses. (At least this is my understanding of it) Is there a workaround for this, ie a key blocker that won't register the e key until it's done defusing?